0000-product.md

```diff
@@ -0,0 +1,501 @@
+is invalid. The default value for grid.product.allow_delete is true. This
+setting is stored using the Sawtooth Settings smart contract, more information
+can be found
+[here](https://sawtooth.hyperledger.org/docs/core/releases/latest/transaction_family_specifications/settings_transaction_family.html).
+
+Validation requirements:
+
+* If a product with product_id does not exist the transaction is invalid.  
+* The signer of the transaction must be an agent in the Pike state and must
+belong to an organization in Pike state, otherwise the transaction is invalid.
+* The owner in the product must match the organization that the agent belongs to, 
+  otherwise the transaction is invalid.  
+* The agent must have the permission “can_delete_product” for the organization 
+  otherwise the transaction is invalid.
+
+The inputs for ProductDeleteAction must include:
+
+* Address of the Agent submitting the transaction 
+* Address of the Organization the Product is being deleted for Address of the Product to be deleted
+
+The outputs for ProductUpdateAction must include:
+
+* Address of the Product to be deleted
+
+### Defined GS1 Properties
+
+An initial set of GS1 properties will be predefined within Grid. Each property
+will have a property definition of the following format:
+
+``` 
+PropertyDefinition( 
+    name="<GS1 Property>",
+    data_type=PropertyDefinition.DataType.STRING, 
+    required=False, 
+    description="A description of the GS1 data.") 
+```
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+
+Each GTIN is derived from three components. A GS1 Global Company Prefix (which
+also identifies the owner of the GTIN), and item reference number, and a check
+digit. This RFC duplicates the organization identification with a Grid-specific
+owner field to tie the product to organizations managed in Pike. In the future,
+it may be good to use the GTIN’s organizational identifier directly.
+
+Currently, Pike does not provide a place to store a GS1 Company Prefix, and the
+permissions around who is authorized to provide this prefix is complicated and
+not a direct extension of how Pike currently works. As such it is out of the
+scope of this RFC.
+
+Instead, Pike’s organization definition will be extended to include metadata,
+similar to Agent’s metadata. Metadata is a Key,Value store of arbitrary data.
+The GS1 Company Prefix will be stored in the metadata under
+“gs1_company_prefix”.
+
+Solving how to properly provide GS1 Company Prefixes to a Pike Organization will
+be solved in a future RFC. This future RFC should integrate the concepts of GLN
+and the smart contract associated with grid gs1 product should be updated to
+reflect that integration.  `
+
+Trade items can include non-material goods, such as services, which will also
+need to be represented within Grid. This is not covered by this RFC.
+
+Some examples of non-material goods: 
+
+- Batch/Lot (LGTIN) or Serialized (SGTIN) 
```

Comment:
Batch/Lot (LGTIN) or Serialized (SGTIN) are still trade items ( material goods) they are just identified at a lower level of specificity than a simple GTIN.  
Logistical Units are also trade items (material goods) 
While we may not be including these initially, it is incorrect to say these are not material goods/trade items.

Reply:
Additionally services and assets are NOT trade items (although assets are material goods...)

Reply:
Perhaps the right approach is to say   - GS1 identifiers not specified for trade items are not included in GRID. (Assets, services) While logistical items (typically identified with an SSCC)  may not be in the PROD RFC, I would not exclude it entirely from GRID

Reply:
✅